setup.py

```diff
@@ -14,16 +14,15 @@
     long_description=open('README.rst', 'r').read(),
     packages=find_packages(),
     install_requires=[
-        'django>=1.8,<1.9a1',
-        'djangorestframework>=3.1',
+        'django>=1.9,<1.10a1',
+        'djangorestframework>=3.3',
```

Comment:
They introduced json field serializer in 3.3, so it is required.

We discussed this with @tjanez and decided to raise some of the requirements (not to the latest version, but to the latest minor version). The reason was that we have to test all combinations. And the second reason is that users can install resolwe in the virtualenv so it won't interfere with others' apps requirements.